Track Buddy/MotionManager.swift

```diff
@@ -72,5 +96,15 @@ class MotionManager: ObservableObject {
         } else if x < maxLeft {
             maxLeft = x
         }
+        
+        let point = CGPoint(x: x, y: z)
+        // Store values over the specified interval for graph tracer line
+        if recentPoints.count >= pointStorageLimit {
+            // TODO: if Deque methods are updated to use @discardableResult we can get rid of _ =
```

Comment:
Nit- TODOs are things you can take action on immediately. They should be cleared out ASAP. You can leave this comment, but I'd recommend removing the "todo" because depending on Apple to change an API isn't exactly actionable